autoload/aergia/tags.vim

```diff
@@ -9,7 +9,7 @@ let s:pattern = s:opening . '[^>]\+' . s:closing " prevent greedy match
 " named tag
 let s:properties = { "name": '', "position": 0, }
 " cmd tag
-let s:cmds = s:opening . '`.\+`=\?[A-Za-z]*' . s:closing
+let s:cmds = s:opening . '`.\{-1,}`=\?[A-Za-z]\{-}' . s:closing
```

Comment:
I see why the first part (until the [A-Za-z] part) is necessary, but I don't think the `\{-}` matters here.

Reply:
You are correct I didnt have alot to test with and just figured this was safer. Looking at it now its safe to omit the \{-}.